tests/test_wave_05.py

```diff
@@ -52,13 +52,15 @@ def test_new_genre_rec_from_empty_friends():
             }
         ]
     }
+    # Act:
 
-    raise Exception("Test needs to be completed.")
-    # *********************************************************************
-    # ****** Complete the Act and Assert Portions of theis tests **********
-    # *********************************************************************
+    recommendations = get_new_rec_by_genre(sonyas_data)
+
+    #assert
+
+    assert len(recommendations) == 0
```

Comment:
Nice! An alternative way is to assert falsy-ness for the case of an empty list:
`assert not recommendations `